src/packages/documents/documents/tracked-reference/repository/document-tracked-reference.server.data.ts

```diff
@@ -61,14 +52,13 @@ export class UmbDocumentTrackedReferenceServerDataSource {
 	 * @return {*}
 	 * @memberof UmbDocumentTrackedReferenceServerDataSource
 	 */
-	async getTrackedReferenceItem(id: string[], skip = 0, take = 20, filterMustBeIsDependency = true) {
+	async getTrackedReferenceItem(id: string[], skip = 0, take = 20) {
```

Comment:
It doesn't look like this method is used. I think we should remove it for simplicity. Then we can also implement the items logic like the rest of the repositories.